locales/de-DE/translation.json

```diff
@@ -81,54 +82,54 @@
   "Error while sending the transaction": "Fehler beim Senden der Transaktion",
   "Enter your password to send the transaction.": "Geben Sie Ihr Passwort ein, um die Transaktion zu senden.",
   "Lock time": "Sperrzeit",
-  "Duration in minutes after which an idle wallet will lock automatically.": "Dauer in Minuten, nach der eine inaktive Brieftasche automatisch gesperrt wird.",
+  "Duration in minutes after which an idle wallet will lock automatically.": "Dauer in Minuten, nach der eine inaktive Wallet automatisch gesperrt wird.",
   "Minutes": "Minuten",
   "Off": "Aus",
   "Theme": "Motiv",
-  "Select the theme and please your eyes.": "Wählen Sie das Motiv und erfreuen Sie Ihre Augen.",
+  "Select the theme and please your eyes.": "Wählen Sie das Motiv und erfreuen Sie sich am Anblick.",
   "Toggle discreet mode (hide amounts).": "Diskreter Modus (Beträge verbergen).",
   "Password requirement": "Passwort erforderlich",
   "Require password confirmation before sending each transaction.": "Vor jeder Transaktion eine Passwortbestätigung verlangen.",
   "Password": "Kennwort",
   "Type your password to change this setting.": "Geben Sie Ihr Passwort ein, um diese Einstellung zu ändern.",
   "Enter": "Eingabe",
   "Custom network settings saved.": "Benutzerdefinierte Netzwerkeinstellungen gespeichert.",
-  "Make sure to always check what is the selected network before sending transactions.": "Stellen Sie sicher, dass Sie immer prüfen, welches Netzwerk ausgewählt ist, bevor Sie Transaktionen senden.",
+  "Make sure to always check what is the selected network before sending transactions.": Prüfen Sie immer welches Netzwerk ausgewählt ist, bevor Sie Transaktionen senden.",
   "Network": "Netzwerk",
-  "Node host": "Knoten Host",
+  "Node host": "Node Host",
   "Explorer API host": "Explorer API Host",
   "Explorer URL": "Explorer-URL",
-  "Wallet list": "Brieftaschenliste",
-  "Current wallet": "Aktuelle Brieftasche",
-  "Wallet name": "Brieftaschen-Name",
-  "Lock current wallet": "Aktuelle Brieftasche sperren",
+  "Wallet list": "Walletliste",
+  "Current wallet": "Aktuelle Wallet",
+  "Wallet name": "Walletname",
+  "Lock current wallet": "Aktuelle Wallet sperren",
   "Show your secret recovery phrase": "Geheime Wiederherstellungsphrase anzeigen",
-  "Remove current wallet": "Aktuelle Brieftasche entfernen",
+  "Remove current wallet": "Aktuelle Wallet entfernen",
```

Comment:
I changed Wallet for Brieftasche because no german dude in crypto says Brieftasche. Older people would say Brieftasche to understand what a Cryptowallet is, but I think every German who is able to create a wallet, wouldn´t say Brieftasche.